webapp/src/components/Controls/Controls.tsx

```diff
@@ -229,23 +230,21 @@ const Controls: React.FC<Props> = ({
             />
           </Box>
           <Box margin={1}>
-            <FormControlLabel
-              control={
-                <Switch
-                  checked={postprocessing.withoutPostprocessing ?? false}
-                  onChange={(_, checked) => handlePostprocessingChange(checked)}
-                />
-              }
-              label="Without PostProcessing"
-            />
+            <Tooltip title="Include or exclude post-processing in predictions and any derived output. This only affects the Exploration Space, and won't affect the smart tags.">
```

Comment:
Nitpick: I think maybe the text shouldn't say "Include" since this the action one takes with this switch is only "Exclude"

I think the text should read:
"Exclude post-processing in predictions and any derived output. This only affects the Exploration Space, and won't affect the smart tags.

Reply:
Makes sense to me! Thank you!
```suggestion
            <Tooltip title="Exclude post-processing in predictions and any derived output. This only affects the Exploration Space, and won't affect the smart tags.">
```